tests/Unit/TestingFramework.hpp

```diff
@@ -0,0 +1,242 @@
+// Distributed under the MIT License.
```

Comment:
You should add this file to `docs/Doxyfile.in` line 86 so that it gets built as part of the documentation.

Reply:
okay, done